cmd/create/accountroles/creators.go

```diff
@@ -17,11 +17,16 @@ type creator interface {
 	buildCommands(*accountRolesCreationInput) (string, error)
 }
 
-func initCreator(managedPolicies bool) creator {
+func initCreator(managedPolicies bool, isHypershift bool) creator {
```

Comment:
The variable renaming is confusing: `--hosted-cp` stored in `args.hostedClusterEnabled` and read as `isHypershift bool`.

I suggest keeping it consistent:
`--hosted-cp` -> `args.hostedCP` -> `hostedCP bool`

Reply:
I agree, changed.

Reply:
/nitpick
I think we still need to change this variable from `isHypershift` to `hostedCP`

Reply:
Right, changed, thanks.